tests/Spec/Nri/Ui/HighlighterToolbar.elm

```diff
@@ -152,24 +234,33 @@ init =
 
 {-| -}
 type Msg
-    = SetTool (Maybe Tag)
+    = FocusAndSelectTag { select : Maybe Tag, focus : Maybe String }
+    | Focused (Result Dom.Error ())
 
 
 {-| -}
 update : Msg -> State -> State
 update msg state =
     case msg of
-        SetTool tag ->
-            { state | currentTool = tag }
+        FocusAndSelectTag { select, focus } ->
+            Tuple.first
```

Comment:
It looks like the cmds will never end up being run here (which is fine, since elm test doesn't keep track of browser focus anyway). I wonder if it's simpler to not have the second part of the tuple at all? rather than create it and then discard it?